Update ReadMe.md (#98)
Laura-Novich-OBW/student-showcase: student-work/2022/Andrea-Parkins/ReadMe.md

# Andrea Parkins #    
_**Technical Writer, writer, translator**_  
Email: akparkins7@gmail.com  
LinkedIn: www.linkedin.com/in/andrea-parkins  
Phone: 053-3981633  
[My Portfolio](https://wix.to/k4BMsS6?)  
____________________  
## About Me ##  

> I currently work as a technical writer with a premier software company. I have a history as an accomplished instructor with exemplary research, problem solving, editing, and technical skills. Throughout my career, I have excelled at simplifying complex concepts and designing processes to support effective student interventions. I have an innate curiosity for technology and how things operate. I consistently seek to use my excellent communication skills to create successful technical communications.  

Reviewer: ```suggestion
I currently work as a technical writer with a premier software company. I am an accomplished instructor with exemplary research, problem solving, editing, and technical skills. Throughout my career, I have excelled at simplifying complex concepts and designing processes to support effective student interventions. I have an innate curiosity for technology and how things operate. I consistently seek to use my excellent communication skills to create successful technical communication.  
```

____________________  
## Skills ##  

> - DITA  

Reviewer: ```suggestion
- DITA XML 
```

- Markdown  
- Madcap Flare  
- Basic UX Writing 

____________________
## Experience ##  

**Technical Writing Intern,** _ServiceNow Israel,_ July. 2022–Present  

> - Created, revise, and maintain technical documentation related to proprietary software and processes  

Reviewer: ```suggestion
- Create, revise, and maintain technical documentation related to proprietary software and processes  
```

Reviewer: If you like the changes please click "Commit Suggestion"
